packages/core/test/controls/numericInputTests.tsx

```diff
@@ -1098,7 +1097,7 @@ describe("<NumericInput>", () => {
             incrementButton.simulate("mousedown", { shiftKey: true });
             expect(component.find("input").prop("value")).to.equal("1.101");
 
-            TestUtils.act(() => {
```

Comment:
Fixes a deprecation in React 18. Act is now exported from React as of `v18.3.1`.

https://react.dev/warnings/react-dom-test-utils#reactdomtestutilsact-warning

> act from react-dom/test-utils has been deprecated in favor of act from react.